package.json

```diff
@@ -61,7 +61,7 @@
         ]
     },
     "resolutions": {
-        "axe-core": "4.0.1",
```

Comment:
Now that we are using @axe-core/puppeteer, I'm wondering whether we don't need the resolution anymore. Earlier axe-puppeteer pinned to 3.x internally, but [@axe-core/puppeteer looks like it has ^4.0.1](https://github.com/dequelabs/axe-core-npm/blob/e6a167f1d7a4e6872280d1c00ff14dddd47188af/packages/puppeteer/package.json#L45).

Reply:
I'll keep resolution to prevent discrepancies when depending on packages change version due to depenabot update